### PR TITLE
Edit attributes of member and event

### LINKED ISSUE #136: Edit Attributes of Member and events

### PR DESCRIPTION
**Pull request by ph-nathan**
Closes #136 

What have you done in this PR?
- Refactored old editCommand to editMemberCommand
- Added editEventCommand, editEventCommandParser
- Fix bug of edited event tags not showing in resultDisplay
- Added tests for editEventCommand
- <Note: This PR hasn't covered #128 and attendance related files>

Dev Testing Steps / Screenshots
image

### CODE REVIEW

File: src/main/java/seedu/ccacommander/logic/Messages.java
Comment (wr1159) [inline src/main/java/seedu/ccacommander/logic/Messages.java:1]: if we're changing the format of member and events can you change the format of Attendance too?

File: src/main/java/seedu/ccacommander/logic/parser/CcaCommanderParser.java [hunk 1]
```diff
@@ -65,8 +66,11 @@ public Command parseCommand(String userInput) throws ParseException {
         case CreateEventCommand.COMMAND_WORD:
             return new CreateEventCommandParser().parse(arguments);
 
-        case EditCommand.COMMAND_WORD:
-            return new EditCommandParser().parse(arguments);
+        case EditMemberCommand.COMMAND_WORD:
+            return new EditMemberCommandParser().parse(arguments);
+
+        case EditEventCommand.COMMAND_WORD:
+            return new EditEventCommandParser().parse(arguments);
```
Comment (wr1159) [inline src/main/java/seedu/ccacommander/logic/parser/CcaCommanderParser.java:73]: As codecov has flagged it already, do you want to a relevant test?

**Review by wr1159 — APPROVED** [review]
LGTM, just a few small comments

File: src/main/java/seedu/ccacommander/logic/parser/CcaCommanderParser.java (hunk 1 above)
Comment (ph-nathan) [reply src/main/java/seedu/ccacommander/logic/parser/CcaCommanderParser.java:73]: Noted! Have added a relevant test.

**Review by dylkaw — APPROVED** [review]
LGTM!

**Pull request merged**